src/commands/Start.ts

```diff
@@ -20,22 +18,21 @@ function error(err: string): void {
 async function run(): Promise<void> {
   try {
     log.info('Starting Enveloping Relay Server process...\n');
-    const contractsConfig: ContractsConfig = config.get('contracts');
-    const appConfig: AppConfig = config.get('app');
-    log.setLevel(appConfig.logLevel);
-    if (!config.has('blockchain.rskNodeUrl')) {
+    const { contracts, app, blockchain } = getServerConfig();
+    log.setLevel(app.logLevel);
+    if (!blockchain.rskNodeUrl) {
       error('missing rskNodeUrl');
     }
-    const trustedVerifiers = contractsConfig.trustedVerifiers;
+    const trustedVerifiers = contracts.trustedVerifiers;
 
     log.debug('runServer() - provider done');
     // config = await resolveServerConfig(conf, provider);
```

Comment:
Do we still need this line to be commented?

Reply:
I removed that part

Reply:
updated